src/main/java/io/hops/hdfs/AbstractFacade.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package io.hops.hdfs;
+
+import java.util.List;
+import javax.persistence.EntityManager;
+
+/**
+ *
+ * @author kerkinos
+ */
+public abstract class AbstractFacade<T> {
```

Comment:
We already have a class called AbstractFacade.java. Can you use that one instead?